### ISSUE TITLE
In ra.chartsjs.js: calculateTotalOnObjectArray is undefined

### ISSUE THREAD

**Issue by squio** [ra-systems/django-slick-reporting#11]
Version: 0.4.0 release

Line 85: 

```js
  let results = calculateTotalOnObjectArray(response.data, seriesColNames);
```
Console.log output:

```txt
Uncaught ReferenceError: calculateTotalOnObjectArray is not defined
    extractDataFromResponse http://localhost/static/slick_reporting/ra.chartsjs.js:85
    createChartObject http://localhost/static/slick_reporting/ra.chartsjs.js:19
    DisplayChart http://localhost/dashboard/sessions/:241
    <anonymous> http://localhost/dashboard/sessions/:232
    jQuery 7
    <anonymous> http://localhost/dashboard/sessions:235
```

The result is that there is no graph rendered at all.

**Comment by RamezIssac** [comment]
Hello @squio 
Thank you for your report !

Both this issue and #10 have been addressed in the branch `develop` in case you wanty to give it a test out and provide any feedback which is very appreciated.

Release will be sometime tomorrow morning nothing new comes up

**Comment by squio** [comment]
Many thanks for the fixes (this and #10), they work great in the new release 0.4.1.
And the time picker is working - I was gonna report that next but you were ahead of me ;-)
